src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -132,8 +132,10 @@ class SearchControls extends Component {
           </div>
           <div className="columns medium-3">
             <label htmlFor="facilityType">Select Facility Type</label>
-            <div tabIndex="1" className={`facility-dropdown-wrapper ${facilityDropdownActive ? 'active' : ''}`} onClick={this.toggleFacilityDropdown} onBlur={() => {this.setState({ facilityDropdownActive: false });}}>
-              {this.renderSelectOptionWithIcon(currentQuery.facilityType)}
+            <div tabIndex="1" className={`facility-dropdown-wrapper ${facilityDropdownActive ? 'active' : ''}`} onClick={this.toggleFacilityDropdown} onBlur={() => {setTimeout(() => {this.setState({ facilityDropdownActive: false });}, 200);}}>
+              <div className="flex-center">
```

Comment:
I think this `setTimeout()` is causing some weirdness. When I try to open the locator the map zooms out every 2ms.